package.json

```diff
@@ -21,6 +21,7 @@
     "check-types": "lerna run check-types",
     "check-spelling": "pyspelling -c .pyspelling.yml -v",
     "docs:build": "lerna run check-readme && lerna run docs:build && ./scripts/prepare-docs.sh",
+    "docs:serve": "yarn --cwd docs install && yarn --cwd docs start",
```

Comment:
`yarn --cwd docs install` will this part be a no-op if dependencies are already installed?

Reply:
Right, it will be almost instant and no-op (`success Already up-to-date.`). I used this approach to have a single command to serve the docs, even the first time.